src/more/more.rs

```diff
@@ -0,0 +1,113 @@
+#![crate_name = "uu_more"]
+
+/*
+ * This file is part of the uutils coreutils package.
+ *
+ * (c) Martin Kysel <code@martinkysel.com>
+ *
+ * For the full copyright and license information, please view the LICENSE file
+ * that was distributed with this source code.
+ */
+
+extern crate getopts;
+
+#[macro_use]
+extern crate uucore;
+
+use getopts::Options;
+use std::io::{stdout, stdin, stderr, Write, Read, Result};
+use std::fs::File;
+
+extern crate nix;
```

Comment:
Blocker: I'm as big a fan of Nix as anyone, but it would be better here to use [termios](https://crates.io/crates/termios) than functionality whose future is tied to a particular linux distribution.